helm/charts/determined/templates/master-permissions.yaml

```diff
@@ -35,6 +35,9 @@ rules:
   - apiGroups: [""]
     resources: ["nodes"]
     verbs: ["list", "watch", "patch"]
+  - apiGroups: ["batch"]
```

Comment:
unneeded changes?

Reply:
we need these now if we are going to operate on jobs